configure.ac

```diff
@@ -868,6 +868,21 @@ case "$host_os" in
 
     AC_DEFINE(OPEN_BSD,,OpenBSD)
     AC_DEFINE(KAME,1,KAME IPv6)
+
+    if test "x${enable_pimd}" != "xno"; then
+      BSDVER=`uname -r`
```

Comment:
All are valid points, will do the change.  Thanks for taking the time to look at this